processor/elasticinframetricsprocessor/testdata/config.yaml

```diff
@@ -1,8 +1,12 @@
-default:
+elasticinframetrics:
+elasticinframetrics/2:
   add_system_metrics: true
+  add_kubernetes_metrics: true
 
-with_system_metrics:
+elasticinframetrics/with_system_metrics:
   add_system_metrics: true
+  add_kubernetes_metrics: false
 
-without_system_metrics:
-  add_system_metrics: false
\ No newline at end of file
+elasticinframetrics/without_system_metrics:
+  add_system_metrics: false
+  add_kubernetes_metrics: false
```

Comment:
Since we are using these metrics to power just the Inventory (part of curated UI's). Should we either both enabled/disabled. Rather than giving different combinations to the user? WDYT ?

Reply:
These are just tests so I believe it does makes sense to test the various combinations unless we enforce the both_enabled/both_disabled rule/idea in the the implementation.
If the latter is the proposal I would suggest we first discuss and validate this with the wider `otel-dev` group.

Reply:
> Should we have either both enabled/disabled

now both `add_system_metrics`/`add_kubernetes_metrics` are enabled by default - https://github.com/elastic/opentelemetry-collector-components/blob/cf215e8e1fe25354459ee6e668f97a1e9f170a94/processor/elasticinframetricsprocessor/factory.go#L25-L28
I agree that it is unlikely that users would need to change the default configuration, but I would keep it configurable to explicitly disable specific remapper if needed

Reply:
I have checked about this in the #otel-dev. Lets see